Bottom: 3726fccaa3e0120257cd7f04253dac6668b8e93b
Top:    23be4a17a741340fc5fc39cf5a60da01be953930
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 16:31:27 +0000

TODO notes


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 5090bb7..06a8e3f 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1,3 +1,12 @@
+/* TODO ITEMS:
+ *
+ * Add PLINE simplification operation to consolidate co-circular segments for reduced geometry output.
+ * Look at whether arc-* intersections can be re-constructed back to original geometry, not fall back to line-line.
+ * Work on snap-rounding any edge which passes through the pixel square containing an any vertex (or intersection).
+ * Avoid self-touching output in contours, where that self-touching instance creates two otherwise distinct contours or holes.
+ *
+ */
+
 #include <stdio.h>
 #include <stdlib.h>
 #include <assert.h>
